Исправление перевода

Paprikar opened the pull request:
Исправил перевод:
- Русалка -> Ундина
- Человеческий Дом -> Земля
- Волшебница Воды -> Водная Фея
- Косички -> Хвостики
- Смущающие замечания -> Смущающие высказывания

Paprikar committed:
Исправил перевод:

- русалка -> ундина

- Человеческий Дом -> Земля

KusanagiMajor commented on [15] Aria TV 2 01 серия.ass:
```diff
@@ -174,7 +174,7 @@ Dialogue: 0,0:09:24.59,0:09:28.59,Default,,0,0,0,,А, Алисия-сан, В-В
+Dialogue: 0,0:09:32.86,0:09:35.76,Default,,0,0,0,,Это мифический зверь с Земли.
 Dialogue: 0,0:09:35.76,0:09:38.56,Default,,0,0,0,,Ты очень мне его напоминаешь.
 Dialogue: 0,0:09:38.56,0:09:39.73,Default,,0,0,0,,Заткнись, Коняшка.
 Dialogue: 0,0:09:39.73,0:09:40.68,Default,,0,0,0,,Коняшка?
```
Коняшка?)))

KusanagiMajor replied:
Надо проверить кто это говорит

Paprikar replied:
Проверил, в контексте всё правильно

Paprikar committed:
Исправил перевод

- Волшебница Воды -> Водная Фея

- Косички -> Хвостики

- Смущающие замечания -> Смущающие высказывания

KusanagiMajor commented on [14] Aria TV 2 Sp 1 серия.ass:
Смущающие **замечания** запрещены!

Paprikar committed:
Удалил двойные пробелы

hotoshi commented on [15] Aria TV 2 01 серия.ass:
199 Все люди, который (которых) я встретила на Акве, собрались вместе.

KusanagiMajor commented on [40] Aria TV 2 26 серия.ass:
Строка 325
"В детстве я хотела стать волшебницей."
И следующий за ней диалог выглядит так будто это игра слов и речь именно про фею.

Это не давало мне покоя, потому я на слух примерно разобрал что они говорят:
> 子供の頃は**妖精**になりたいともってました
> **妖精**?
> はい、色んなところお飛び回って、みんあお幸せするような
> へへ、成れるんじゃない？
> おおえ？
> あかりちゃんももうすぐ**妖精**さんに
> おお

Если пропустить это через deepl то получится:
> When I was a child, I wanted to be a **fairy**.
> **Fairy**?
> Yes, I wanted to fly around and make people happy.
> Heh, you can do that, can't you?
> Oh, yeah?
> Akari, you'll be a **fairy** soon, too.
> Oh!

KusanagiMajor replied:
И google и deepl переводят 妖精 (ようせい) как fairy, но куда важнее то, что это то же слово что они употребляют, когда говорят про водных фей - 水の妖精, где 水 это вода, а の это частица показывающая принадлежность одного к другому (откуда и получается "фея воды", или же "водная фея").

В общем важно использовать то же слово, потому как конец диалога это именно намёк на то что она станет Водной Феей.

hotoshi commented on [18] Aria TV 2 04 серия.ass:
191 Я буду грести назад.
192 Грести назад?

Может "Задом наперёд"?

Paprikar replied:
@KusanagiMajor ты уже проверил с видеорядом?

KusanagiMajor replied:
@Paprikar проверил, но там и по диалогу было понятно что они торопятся.

Paprikar committed:
Исправил перевод

KusanagiMajor commented on [16] Aria TV 2 02 серия.ass:
```diff
@@ -194,7 +194,7 @@ Dialogue: 0,0:10:06.67,0:10:08.09,Default,,0,0,0,,Вот это да.
 Dialogue: 0,0:10:11.13,0:10:14.05,Default,,0,0,0,,и я не могла заставить себя прийти сюда...
 Dialogue: 0,0:10:17.76,0:10:18.89,Default,,0,0,0,,Это точно.
-Dialogue: 0,0:10:18.89,0:10:24.81,Default,,0,0,0,,Фактически, это кафе открылось более чем 580 лет назад в Венеции на Человеческом Доме,
+Dialogue: 0,0:10:18.89,0:10:24.81,Default,,0,0,0,,Фактически, это кафе открылось более чем 580 лет назад в Венеции на Земле,
```
в Земной Венеции

Paprikar replied:
+

Paprikar replied:
:heavy_check_mark:

KusanagiMajor commented on [16] Aria TV 2 02 серия.ass:
```diff
@@ -252,7 +252,7 @@ Dialogue: 0,0:14:08.62,0:14:12.66,Default,,0,0,0,,Вы, наверное, оче
 Dialogue: 0,0:14:14.00,0:14:16.84,Default,,0,0,0,,"Это самое лучшее место в мире для рисования."
 Dialogue: 0,0:14:16.84,0:14:17.64,Default,,0,0,0,,Э?
-Dialogue: 0,0:14:17.64,0:14:24.88,Default,,0,0,0,,Это то, что сказал Наполеон о площади Сан Марко давным-давно на Человеческом Доме.
+Dialogue: 0,0:14:17.64,0:14:24.88,Default,,0,0,0,,Это то, что сказал Наполеон о площади Сан Марко давным-давно на Земле.
```
Святого Марко (возможно в других местах тоже нужно заменить)

Paprikar replied:
+

Paprikar replied:
:heavy_check_mark:

KusanagiMajor commented on [17] Aria TV 2 03 серия.ass:
```diff
@@ -30,9 +30,9 @@ Dialogue: 0,0:00:49.28,0:00:52.94,Default,,0,0,0,,Да. А может холод
 Dialogue: 0,0:00:54.91,0:00:56.27,Default,,0,0,0,,Тебе не кажется, что твои обвинения малость притянуты?
 Dialogue: 0,0:00:56.27,0:01:01.08,Default,,0,0,0,,Нет! Это точно он виноват! Я никогда не забуду боль в моих обмороженных плечах!
-Dialogue: 0,0:01:01.08,0:01:06.67,Default,,0,0,0,,На Человеческом Доме погода всегда стабильна, но мне больше нравится Аква.
+Dialogue: 0,0:01:01.08,0:01:06.67,Default,,0,0,0,,На Земле погода всегда стабильна, но мне больше нравится Аква.
```
いつもお天気だったけど

"всегда была хорошая погода"
или
"погода всегда была хорошая"

Paprikar replied:
+

Paprikar replied:
:heavy_check_mark:

KusanagiMajor commented on [17] Aria TV 2 03 серия.ass:
```diff
@@ -195,10 +195,10 @@ Dialogue: 0,0:11:53.70,0:11:58.03,Default,,0,0,0,,Новая Венеция оч
 Dialogue: 0,0:12:06.66,0:12:11.63,Default,,0,0,0,,Мы похожи на исследователей звёзд, ищущих метеоритный дождь!
 Dialogue: 0,0:12:13.71,0:12:17.01,Default,,0,0,0,,Точно! А Ал-кун это командир нашей маленькой экспедиции?
-Dialogue: 0,0:12:17.01,0:12:19.02,Default,,0,0,0,,Вы двое, смущающие замечания запрещены!
+Dialogue: 0,0:12:17.01,0:12:19.02,Default,,0,0,0,,Вы двое, Смущающие высказывания запрещены!
```
Настолько смущающие что с большой буквы?)

Paprikar replied:
+

Paprikar replied:
:heavy_check_mark:

KusanagiMajor commented on [17] Aria TV 2 03 серия.ass:
```diff
@@ -293,7 +293,7 @@ Dialogue: 0,0:21:56.17,0:22:01.05,Default,,0,0,0,,Очень редко, но и
 Dialogue: 0,0:22:12.64,0:22:13.72,Default,,0,0,0,,С...
-Dialogue: 0,0:22:13.72,0:22:15.48,Default,,0,0,0,,Смущающие замечания запрещены!
+Dialogue: 0,0:22:13.72,0:22:15.48,Default,,0,0,0,,Смущающие высказывания запрещены!
 Dialogue: 0,0:22:17.44,0:22:23.73,Default,,0,0,0,,{\i1}Благодаря Ал-чану, сегодняшняя ночь метеоритного дождя прошла в тишине.
```
"Ал-чану" это если что не ошибка, так не только к девочкам но и к детям обращаются (хоть он их и старше).
В аниме она так и говорит "Ал-чан".

Paprikar replied:
Так мне это исправлять или как?)))

KusanagiMajor replied:
Нет, просто пометил чтоб вы с Димой случайно не исправили.
Можно в резолвд.

Paprikar replied:
Я про то, не нужно ли заменить на "Ал-чан"?

KusanagiMajor replied:
Нене, не стоит

KusanagiMajor commented on [23] Aria TV 2 09 серия.ass:
```diff
@@ -52,11 +52,11 @@ Dialogue: 0,0:03:27.09,0:03:29.63,Default,,0,0,0,,Если Вы посмотри
 Dialogue: 0,0:03:31.93,0:03:37.07,Default,,0,0,0,,в этом городе повсюду есть\Nмножество изображений ромашек.
 Dialogue: 0,0:03:38.40,0:03:44.01,Default,,0,0,0,,Говорится, что этот цветок был\Nпривезён Марко Поло из Китая...
-Dialogue: 0,0:03:44.27,0:03:47.98,Default,,0,0,0,,и быстро стал очень популярным\Nв Венеции на Человеческом Доме.
+Dialogue: 0,0:03:44.27,0:03:47.98,Default,,0,0,0,,и быстро стал очень популярным\Nв Венеции на Земле.
```
Земной Венеции

Paprikar replied:
:heavy_check_mark:

KusanagiMajor commented on [32] Aria TV 2 18 серия.ass:
```diff
@@ -293,7 +293,7 @@ Dialogue: 0,0:20:41.69,0:20:50.52,Song,,0,0,0,,Передо мною весь г
 Dialogue: 0,0:20:50.52,0:20:59.65,Song,,0,0,0,,Ветер волосы нежно гладит мои.
 Dialogue: 0,0:20:54.11,0:20:55.90,Default,,0,0,0,,Айка-чан, как здорово!
-Dialogue: 0,0:20:56.49,0:20:58.32,Default,,0,0,0,,Да, но смущающие замечания запрещены!
+Dialogue: 0,0:20:56.49,0:20:58.32,Default,,0,0,0,,Да, но Смущающие высказывания запрещены!
```
Опять слишком смущающие высказывания, что аж с большой буквы

Paprikar replied:
:heavy_check_mark:

KusanagiMajor commented on [34] Aria TV 2 20 серия.ass:
```diff
@@ -70,7 +70,7 @@ Dialogue: 0,0:05:47.33,0:05:50.16,Default,,0,0,0,,Нет лучшего сред
 Dialogue: 0,0:05:58.50,0:05:59.30,Default,,0,0,0,,Да.
 Dialogue: 0,0:06:00.92,0:06:04.05,Default,,0,0,0,,Что? Эта женщина ушла.
-Dialogue: 0,0:06:04.05,0:06:11.39,Default,,0,0,0,,Эти колонны всегда были олицетворением города, даже в Венеции на Человеческом доме.
+Dialogue: 0,0:06:04.05,0:06:11.39,Default,,0,0,0,,Эти колонны всегда были олицетворением города, даже в Венеции на Земле.
```
Земной Венеции

Paprikar replied:
:heavy_check_mark:

KusanagiMajor commented on [38] Aria TV 2 24 серия.ass:
```diff
@@ -125,19 +125,19 @@ Dialogue: 0,0:07:08.74,0:07:12.70,Default,,0,0,0,,Я думаю, что нам 
 Dialogue: 0,0:07:15.67,0:07:17.42,Default,,0,0,0,,Прозвище Алисии-сан...
-Dialogue: 0,0:07:18.09,0:07:21.86,Default,,0,0,0,,Белый Снег. Белая Волшебница.
+Dialogue: 0,0:07:18.09,0:07:21.86,Default,,0,0,0,,Белый Снег. Белая Фея.
 Dialogue: 0,0:07:21.86,0:07:24.19,Default,,0,0,0,,Атена-семпай - Сирена.
```
Афина-семпай (надо проверить в других местах)

![](https://img33.imgslib.link//manga/aria/chapters/5-22/11.png)

KusanagiMajor replied:
Скрин из переведённой манги это так себе пруф конечно, но я считаю перевод манги очень удачным в плане адаптации, потому что он действительно притно воспринимается без всякого кринжа, так что беру его за эталон.

Paprikar replied:
:heavy_check_mark:

Paprikar committed:
Исправил перевод

- Венеция на Земле -> Земная Венеция

Paprikar committed:
Исправил перевод

- площадь Сан Марко -> площадь Святого Марко

Paprikar committed:
Мелкие исправления перевода

Paprikar committed:
Исправления перевода

- Атена -> Афина

KusanagiMajor commented:
Ещё меня раздражает что субтитры почему-то цвета мочи.
Думаю после наших правок мы могли бы вернуть им благородный белый цвет.

Нужно просто заменить
> Default,Arial,60,&H000BFFFD

на
> Default,Arial,60,&H00FFFFFF

в каждой серии

KusanagiMajor reviewed (changes requested):
Ещё меня раздражает что субтитры почему-то цвета мочи.
Думаю после наших правок мы могли бы вернуть им благородный белый цвет.

Нужно просто заменить
> Default,Arial,60,&H000BFFFD

на
> Default,Arial,60,&H00FFFFFF

в каждой серии

Paprikar committed:
Правки шрифтов

https://github.com/Paprikar/aria-2-subs/pull/1#pullrequestreview-1471230747

Paprikar commented:
> Ещё меня раздражает что субтитры почему-то цвета мочи. Думаю после наших правок мы могли бы вернуть им благородный белый цвет.
> 
> Нужно просто заменить
> 
> > Default,Arial,60,&H000BFFFD
> 
> на
> 
> > Default,Arial,60,&H00FFFFFF
> 
> в каждой серии

✔️

KusanagiMajor commented on [14] Aria TV 2 Sp 1 серия.ass:
```diff
@@ -11,7 +11,7 @@ ScaledBorderAndShadow: yes
 [V4+ Styles]
 Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
-Style: Default,Arial,60,&H000BFFFD,&H00000000,&H00000000,&H00000000,-1,0,0,0,100,100,0,0,1,4,0,2,30,30,30,204
+Style: Default,Arial,60,&H00FFFFFF,&H00000000,&H00000000,&H00000000,-1,0,0,0,100,100,0,0,1,4,0,2,30,30,30,204
```
почему гит видит разницу начиная с H00... а не с 0B... ?

Paprikar replied:
Хз, видимо подсвечивает диффу по словам а не по символам (коды символов - правильные)

KusanagiMajor commented on [19] Aria TV 2 05 серия.ass:
Строка 53 (и далее)
Oinari-san - это не жареный тофу, это инари суши

Жареный тофу:
![image](https://github.com/Paprikar/aria-2-subs/assets/67715102/dfb4c7de-ff7d-4b93-8650-9941e89e9b02)

Инари суши:
![](https://susi-college.com/wp-content/uploads/2017/12/inari-sushi_4.jpg)

Технически это суши в мешочках из обжаренного тофу, китсуне любят тофу, и потому им нравятся инари, так как они завёрнуты в тофу. Проблема только в том, что в сабах всё перевели как жаренный тофу, хотя они там покупают инари.

В манге перевели так:
![](https://img33.imgslib.link//manga/aria/chapters/1-4/07.png)
![](https://img33.imgslib.link//manga/aria/chapters/1-4/33.png)

Итого надо заменить:

> Oinari-san (inarizushi), "inari" - жареный соевый творог: в японском фольклоре считается, что божественные лисы очень его любят.
на
> Oinari-san (inarizushi), "inari" - разновидность суши, завёрнутых в обжаренный тофу.

далее во всей серии заменить "жареный тофу" на "инари", кроме вот этой строчки:

> С давних пор, китсуне очень любят жареный тофу.

Paprikar replied:
✅

KusanagiMajor commented on [19] Aria TV 2 05 серия.ass:
Строка 87 (и далее)
грибной дождь -> слепой дождь

Paprikar replied:
✅

Paprikar replied:
☑️

hotoshi commented on [29] Aria TV 2 15 серия.ass:
```diff
@@ -68,10 +68,10 @@ Dialogue: 0,0:04:36.09,0:04:42.77,Default,,0,0,0,,Но если долго см
 Dialogue: 0,0:04:42.77,0:04:45.81,Default,,0,0,0,,И становится немного одиноко.
 Dialogue: 0,0:04:45.81,0:04:47.56,Default,,0,0,0,,Ко-сич-ки!
```
Ко-сич-ки! -> Хво-сти-ки!

Paprikar replied:
✅

Paprikar committed:
Исправления перевода

- жареный тофу -> инари

Paprikar committed:
Исправления перевода

- грибной дождь -> слепой дождь

Paprikar committed:
Исправления перевода

- Ко-сич-ки -> Хво-сти-ки

KusanagiMajor reviewed (approved):
С кайфом засмотрим теперь

hotoshi commented on [36] Aria TV 2 22 серия.ass:
220 Кэрри со свиными котлетами -> Карри со свиными котлетами.

Paprikar replied:
✅

hotoshi commented on [19] Aria TV 2 05 серия.ass:
```diff
@@ -120,16 +120,16 @@ Dialogue: 0,0:10:37.34,0:10:38.85,Default,,0,0,0,,Где ты была?
 Dialogue: 0,0:10:43.78,0:10:45.62,Default,,0,0,0,,Что? Правда?
-Dialogue: 0,0:10:45.62,0:10:49.46,Default,,0,0,0,,Да! И один из них взял мой жареный тофу!
+Dialogue: 0,0:10:45.62,0:10:49.46,Default,,0,0,0,,Да! И один из них взял мои инари!
 Dialogue: 0,0:10:49.46,0:10:54.30,Default,,0,0,0,,С давних пор, китсуне очень любят жареный тофу.
```
С давних пор, китсуне очень любят жареный тофу -> С давних пор, китсуне очень любят инари?

Paprikar committed:
Исправления перевода

- мелкие исправления

- так так -> ну и ну

Paprikar committed:
Исправления перевода

- кэрри -> карри
